T24442

tchx84 opened the pull request:
https://phabricator.endlessm.com/T24442

tchx84 committed:
System: highlight selected icon

Re-use the PopupMenu we used in all other toolboxes
so we achieve the exact same results.

https://phabricator.endlessm.com/T24442

xjuan commented:
Looks good, the only thing is that the selected cursor does not have the focus when you first open the popup, so we either need to sync the focus with highlighted item or simply make sure items can not have focus

I think I prefer the later

tchx84 committed:
src: PopupMenu should also focus selected item

This way, while using arrows to navigate,
both highlight  and focus moves together,
making it easier to use and visually more
consistent.

https://phabricator.endlessm.com/T24442

xjuan commented:
Looks good!

CR Pass
